File: q2864/q2864.java
class Solution {
    public String maximumOddBinaryNumber(String s) {
        int numOfOnes = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == '1')
                numOfOnes++;
        }

        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < numOfOnes - 1; i++)
            sb.append("1");
        for (int i = 0; i < s.length() - numOfOnes; i++)
            sb.append("0");
        sb.append("1");
        return sb.toString();
    }
}

File: q2864/q2864.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        numOfOnes = 0 
        for c in s: 
            if c == '1': 
                numOfOnes += 1
        
        res = "1" * (numOfOnes-1) + "0" * (len(s) - numOfOnes) + "1"

        return res